test/events_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Fable.EventsTest do
+  use ExUnit.Case, async: true
+  import Ecto.Query
+  alias Fable.Test.Aggregate
+
+  defmodule Events do
+    use Fable.Events, repo: Repo
+  end
+
+  describe "emit/2" do
+    test "returns a zero arity callback with an query as first argument" do
+      query = from a in Aggregate, where: a.id > 0
```

Comment:
Repo.serial was designed to work with queryables mostly because I use that function within my code base for other stuff. For this particular purpose, we definitely do NOT want to support queryables, only actual aggregates. The query you have here locks the entire table, which is not ideal.

Reply:
Yeah, didn't really think of that. I just read the `repo.one` and thought it would just affect that one result. I could see using a queryable being useful, but I doubt there's a good way to make it work. Adding `limit: 1` sounds terrible and otherwise we cannot really ensure only one row is selected.